code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -151,6 +151,9 @@
 	for(var/plane_key in plane_masters)
 		var/atom/movable/screen/plane_master/plane = plane_masters[plane_key]
 		if(!plane.allows_offsetting)
```

Comment:
`allows_offsetting` needs to be changed from `TRUE`/`FALSE` to a set of flags called `offsetting_flags` (in this case `BLOCKS_OFFSETTING` and `{a flag that indicates we want to offset relays whose name I cannot come up with}`)

This is all ultimately an optimization and there are some cases where we do not want this code to run (FOV currently is the sole example but I could see more coming up)

Reply:
I'mma go for `OFFSET_RELAYS` tentatively while I work this out, mayyyybe `BLOCKS_PLANE_OFFSETTING` to be more specific about _what_ if blocks offsetting for to be more clear in relation to offsetting relays

Gonna think about it more while coding it up though